plugins/modules/dcnm_network.py

```diff
@@ -2223,6 +2533,11 @@ def validate_input(self):
 
         else:
 
+            if self.dcnm_version > 11:
```

Comment:
Please add a comment indicating why this is needed to avoid future confusion

Reply:
Changes done as suggested

Reply:
Changes done as suggested